pkg/cli/init.go

```diff
@@ -46,7 +46,7 @@ func (c *cli) newInitCmd() *cobra.Command {
 	cmd.Flags().String(projectVersionFlag, c.defaultProjectVersion,
 		fmt.Sprintf("project version, possible values: (%s)", strings.Join(c.getAvailableProjectVersions(), ", ")))
 	// The --plugins flag can only be called to init projects v2+.
-	if c.projectVersion != config.Version1 {
+	if c.projectVersion != config.Version2 {
```

Comment:
I think this is ok, since project version 2 isn't the default anymore and doesn't have the concept of plugins in its spec.